libs/cde-visualization/src/lib/components/histogram/histogram.component.html

```diff
@@ -20,10 +20,16 @@
           [cdkConnectedOverlayOpen]="infoOpen"
           [cdkConnectedOverlayOffsetX]="5"
         >
-          <div>
-            The graph shows a histogram of cell-to-nearest-anchor cell distance distributions categorized by each cell
-            type in the dataset.
-          </div>
+          <hra-tooltip-card
+            small
+            [content]="[
```

Comment:
Move content into a readonly variable on the component class and use that instead of embedding complex objects in the html